basyx.aasrepository/basyx.aasrepository-backend-mongodb/src/test/java/org/eclipse/digitaltwin/basyx/aasrepository/backend/mongodb/TestMongoDBAasRepository.java

```diff
@@ -92,6 +92,10 @@ protected AasService getAasServiceWithThumbnail() throws IOException {
 
 		FileMetadata defaultThumbnail = new FileMetadata("dummyImgA.jpeg", "", createDummyImageIS_A());
 		
+		if(fileRepository.exists(defaultThumbnail.getFileName())) {
```

Comment:
What is the purpose of this?

Reply:
If someone tries to save a file, that already exists, an FileHandlingException is thrown in the save function of the FileRepository. If this happens in the MongoDB part (in any repsoitory), the only way to solve this I found was to delete the existing file and save the new. If this is the wrong thinking and there is a better way, I am willing to change it.